gtm/src/template-parameters.json

```diff
@@ -0,0 +1,404 @@
+[
+  {
+    "macrosInSelect": false,
+    "selectItems": [
+      {
+        "displayValue": "Init",
+        "value": "init"
+      },
+      {
+        "displayValue": "Clicked Object IDs",
+        "value": "clickedObjectIDs"
+      },
+      {
+        "displayValue": "Clicked Object IDs After Search",
+        "value": "clickedObjectIDsAfterSearch"
+      },
+      {
+        "displayValue": "Clicked Filters",
+        "value": "clickedFilters"
+      },
+      {
+        "displayValue": "Converted Object IDs",
+        "value": "convertedObjectIDs"
+      },
+      {
+        "displayValue": "Converted Objects IDs After Search",
+        "value": "convertedObjectIDsAfterSearch"
+      },
+      {
+        "displayValue": "Converted Filters",
+        "value": "convertedFilters"
+      },
+      {
+        "displayValue": "Viewed Object IDs",
+        "value": "viewedObjectIDs"
+      },
+      {
+        "displayValue": "Viewed Filters",
+        "value": "viewedFilters"
+      }
+    ],
+    "displayName": "Event Type",
+    "simpleValueType": true,
+    "name": "eventType",
```

Comment:
it would be more accurate to call this method, since it can also be `'init'`, which is not an event for insights.
Event type in the library is an enum: https://github.com/algolia/search-insights.js/blob/927e74e9b2dcdbb3ebe35fae7d30422987d8a33c/lib/_sendEvent.ts#L5

Reply:
Thanks!